src/lib.rs

```diff
@@ -74,12 +81,25 @@ fn json_set(ctx: &Context, args: Vec<String>) -> RedisResult {
         (None, Some(SetOptions::AlreadyExists)) => Ok(().into()),
         (None, _) => {
             let doc = RedisJSON::from_str(&value)?;
-            key.set_value(&REDIS_JSON_TYPE, doc)?;
-            REDIS_OK
+            if path == "$" {
+                key.set_value(&REDIS_JSON_TYPE, doc)?;
+                REDIS_OK
+            } else {
+                Err("ERR new objects must be created at the root".into())
+            }
         }
     }
 }
 
+///
+/// JSON.GET <key>
+//         [INDENT indentation-string]
```

Comment:
Doc comment is inconsistent -- some lines start with `//` instead of `///`

Reply:
fixed